### DISCUSSION TITLE
this function will be removed in lspconfig version 0.2.0

### DISCUSSION THREAD

**Discussion by binary4cat** [LazyVim/LazyVim#538] [Q&A]
I am sorry to hear that I could not find any helpful information through Google search and I am not sure which plugin caused the issue. I am reaching out here to seek for help from others as I use LazyVim, and greatly appreciate any assistance. My English is not good, so please excuse me if there are any language issues.

1. Enter nvim, this warning will appear
![image](https://user-images.githubusercontent.com/12838659/230325213-23c624bc-b69a-4be3-ab29-2481ff2d3de7.png)

2. Then press Enter to dismiss the prompt and continue to enter the desired file through the "Recent files" menu, one line of content will be hidden.
![image](https://user-images.githubusercontent.com/12838659/230325749-986ea0b6-30cf-4c81-a16c-0b389b2a8a2d.png)

**Comment by binary4cat** [comment]
I found the problem, which is a deprecated LSP configuration that caused this prompt. Because that line happens to be out of sight, I couldn't figure out which deprecated configuration it was. I will continue to search for why that line was obscured.

**Comment by I-Rakesh** [comment]
i got the same error is did change my config to ts_ls from tsserver but i am still getting same error

<img width="1792" alt="Screenshot 2024-09-07 at 09 20 03" src="https://github.com/user-attachments/assets/35c5000f-65c1-4563-8cf5-bf47ae033d65">

<img width="1792" alt="Screenshot 2024-09-07 at 09 21 57" src="https://github.com/user-attachments/assets/556ca858-acc3-4b8f-9a4e-02a11c1bff7b">
